sphinx_antsibull_ext/antsibull-minimal.css

```diff
@@ -1,2 +1,28 @@
 @charset "UTF-8";
-/* Copyright (c) Ansible and contributors */table.documentation-table{border-bottom:1px solid #000;border-right:1px solid #000}table.documentation-table td,table.documentation-table th{padding:4px;border-left:1px solid #000;border-top:1px solid #000}table.documentation-table td.elbow-placeholder{border-top:0;width:30px;min-width:30px}table.documentation-table td{vertical-align:top}table.documentation-table td:first-child{white-space:nowrap}table.documentation-table tr .ansibleOptionLink{display:inline-block;visibility:hidden}table.documentation-table tr .ansibleOptionLink:after{content:"🔗"}table.documentation-table tr:hover .ansibleOptionLink:after{visibility:visible}
\ No newline at end of file
+/* Copyright (c) Ansible and contributors */
+table.documentation-table {
```

Comment:
It looks something didn't work with the build process, since the minification step wasn't applied.

Reply:
should the minification be done in the commit or is it done once the commit is done by some build ?

Reply:
I would do it in the commit, but if for some reason it doesn't work for you, I can do it in a follow-up commit before merging.